storageFormats/eventMetadata.cc

```diff
@@ -316,6 +316,22 @@ eventMetadata* rpwa::eventMetadata::merge(const vector<const eventMetadata*>& in
 	if(mergeDiffMeta) {
 		mergee->setMultibinBoundaries(mergedmultibinBoundaries);
 	}
+
+	for (const auto& input : inputData) {
```

Comment:
Why make a separate loop here? Why not include it in the loop over the input data above?